embulk-core/src/main/sh/selfrun.sh

```diff
@@ -50,6 +50,38 @@ while true; do
     esac
 done
 
+java_fullversion=`java -fullversion 2>&1`
```

Comment:
I got the following error. (macOS)
Probably, You can't use `2>&1` inside back quote.


```bash
% pkg/embulk-0.9.9-SNAPSHOT.jar
pkg/embulk-0.9.9-SNAPSHOT.jar: line 2: 1: command not found
java full version "1.8.0_121-b13"
```

```bash
sh -x  pkg/embulk-0.9.9-SNAPSHOT.jar
+ :
++ 1
++ java -fullversion '2^'
pkg/embulk-0.9.9-SNAPSHOT.jar: line 2: 1: command not found
```

Reply:
@hiroyuki-sato Hmm, really? It was working fine on my environment...

Reply:
Ah, but now I confirmed not working... Will have a look.

Reply:
It's actually just imported from your example at #1020, and simplified. Do you have any information?

Reply:
The shell script by itself still works. It looks that the error has come from the .BAT file part, that should have been skipped.

Reply:
I'll take a look it.

Reply:
Sorry, This part seems to work fine. It seems another part.

This following is part for windows platform, but It seems to execute in `/bin/sh`.

```
for /f "usebackq delims=" %%w in (`java -fullversion 2^>^&1`) do set java_fullversion=%%w
```